Assets/LeapMotionModules/Interaction/Examples/Basic Example/ExampleScene.unity

```diff
@@ -13,7 +13,7 @@ OcclusionCullingSettings:
 --- !u!104 &2
```

Comment:
Should probably change this scene back (or maybe incorporate your new oblong shapes into the original diversity of shapes).